### PR TITLE
mcobbett 1341 ecosystem1 setup on planb cluster

### PR DESCRIPTION
**Pull request by techcobweb**
- Adding ecosystem1 to the planb infrastructure
- changing ecosystem1 to galasa-ecosystem1
- catch github up with the changes we made manually to set up ecosystem1

### CODE REVIEW

File: infrastructure/galasa-plan-b-lon02/galasa-ecosystem1/admin-role.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#
```
Comment (eamansour) [inline infrastructure/galasa-plan-b-lon02/galasa-ecosystem1/admin-role.yaml:1]: We've already got an admin role checked in, so this can be removed.

**Comment by galasa-team** [comment]
Build successful

**Review by eamansour — DISMISSED** [review]
Approved assuming the conflict will be resolved

**Review by eamansour — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**